### DISCUSSION TITLE
SIM8200EA-M2 5G Router connection to the network

### DISCUSSION THREAD

**Discussion by smtsotso** [srsran/srsRAN_Project#802] [General Help]
Greetings,
I have connected the SIM8200EA-M2 5G Router to the SRS network & I cannot access the internet from the phone (connected to router) . What is the issue and how can I solve this?

**Comment by pgawlowicz** [comment]
Did you setup routing, IP forwarding and masquerading correctly?

**Reply by smtsotso** [reply]
Yes, I did. But still. What could possibly be a problem?

**Reply by pgawlowicz** [reply]
Could you provide your configs and logs?

**Reply by smtsotso** [reply]
![logs](https://github.com/user-attachments/assets/8b92dbad-408f-4004-a3c8-1fc10b0adfb5)

**Reply by smtsotso** [reply]
![Screenshot from 2024-09-20 10-04-30](https://github.com/user-attachments/assets/348b9a97-fb6f-4217-a12f-a6409da8b912)

**Reply by pgawlowicz** [reply]
did you set the slicing configuration correctly for each UE in the open5gs, seems there is an issue.

**Reply by pgawlowicz** [reply]
@smtsotso any update on this issue?
